src/overlays/actors/ovl_En_Ru2/z_en_ru2.c

```diff
@@ -1,14 +1,78 @@
+/*
+ * File: z_en_ru2.c
+ * Overlay: En_Ru2
+ * Description:
+ */
+
 #include "z_en_ru2.h"
 
+#include <vt.h>
+
 #define ROOM 0x00
 #define FLAGS 0x00000010
 
-void EnRu2_Init(EnRu2* this, GlobalContext* globalCtx);
-void EnRu2_Destroy(EnRu2* this, GlobalContext* globalCtx);
-void EnRu2_Update(EnRu2* this, GlobalContext* globalCtx);
-void EnRu2_Draw(EnRu2* this, GlobalContext* globalCtx);
+static void EnRu2_Init(EnRu2* this, GlobalContext* globalCtx);
+static void EnRu2_Destroy(EnRu2* this, GlobalContext* globalCtx);
+static void EnRu2_Update(EnRu2* this, GlobalContext* globalCtx);
+static void EnRu2_Draw(EnRu2* this, GlobalContext* globalCtx);
+
+static void func_80AF2CB4(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF2CD4(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF2CF4(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF2D2C(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF2D6C(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF2DAC(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF2DEC(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF3144(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF3174(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF31C8(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF3604(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF3624(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF366C(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF36AC(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF3BC8(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF3C04(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF3C64(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF3CB8(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF3D0C(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF3D60(EnRu2* this, GlobalContext* globalCtx);
+
+static void func_80AF3F14(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF3F20(EnRu2* this, GlobalContext* globalCtx);
+static void func_80AF321C(EnRu2* this, GlobalContext* globalCtx);
+
+static void func_80AF2AB4(EnRu2* this, GlobalContext* globalCtx);
+
+static ColliderCylinderInit D_80AF40E0 = {
+    0x0A, 0x00,       0x09, 0x00, 0x01,   0x00,   0x00,       0x00,   0x00,   0x00,   0x00,
+    0x00, 0x00000000, 0x00, 0x00, 0x00,   0x00,   0x00000080, 0x00,   0x00,   0x00,   0x00,
+    0x00, 0x01,       0x00, 0x00, 0x001E, 0x0064, 0x0000,     0x0000, 0x0000, 0x0000,
+};
+
+static u32 D_80AF410C[] = {
```

Comment:
I think we should stick to UNK_PTR for all segment addresses (that we dont know about) moving forward. It will make it alot easier to find them in the future when we know more about these data types and we want to replace them. Same goes for the cutscene data in the other c file

Reply:
Those aren't "segment addresses" since they are from the actor overlay itself.

Reply:
its an array of segment addresses

Reply:
Ah right, but the cutscene data can stay as u32 or u8 arrays for now.

Reply:
gotcha. going off of our convo in the other pr, this should be UNK_TYPE instead then

Reply:
If you mean the array of segment addresses then no, that should be an UNK_PTR array since it's an array of pointers.